distribution/lib/Standard/Table/0.0.0-dev/src/Errors.enso

```diff
@@ -499,6 +499,19 @@ type Unmatched_Columns
     to_display_text self =
         "The following columns were not present in some of the provided tables: " + (self.column_names.map (n -> "["+n+"]") . join ", ") + "."
 
+type Unused_Columns
```

Comment:
Could we use `Unexpected_Extra_Columns`?

Reply:
Yes :) Didn't spot that one